MapsetVerifier.Checks/Taiko/Timing/CheckDoubleBarlines.cs

```diff
@@ -0,0 +1,134 @@
+﻿using MapsetVerifier.Framework.Objects;
+using MapsetVerifier.Framework.Objects.Attributes;
+using MapsetVerifier.Framework.Objects.Metadata;
+using MapsetVerifier.Parser.Objects;
+using MapsetVerifier.Parser.Objects.TimingLines;
+using MapsetVerifier.Parser.Statics;
+
+using static MapsetVerifier.Checks.Utils.GeneralUtils;
+
+namespace MapsetVerifier.Checks.Taiko.Timing
+{
+    [Check]
+    public class CheckDoubleBarlines : BeatmapCheck
+    {
+        private const string Problem = nameof(Issue.Level.Problem);
+        private const string Warning = nameof(Issue.Level.Warning);
+        private const string RoundingErrorWarning = "roundingErrorWarning";
+
+        public override CheckMetadata GetMetadata() =>
+            new BeatmapCheckMetadata()
+            {
+                Author = "Hivie, Phob",
+                Category = "Timing",
+                Message = "Double barlines",
+                Modes = [Beatmap.Mode.Taiko],
```

Comment:
I'd assume this check and (some of or all?) of the other barlines checks may be relevant in Mania as well?